### PR TITLE
Add note about JavaScript and HTML file naming

### PR DESCRIPTION
**Pull request by scottcorgan**
Closes https://github.com/desiringgod/guides/issues/44

### CODE REVIEW

**Comment by scottcorgan** [comment]
@hchasey assigning you since you made the call on the issue 👍

**Comment by hchasey** [comment]
Should this be added to https://github.com/desiringgod/guides/tree/master/style/sass and https://github.com/desiringgod/guides/tree/master/style/html as well @scottcorgan ?

**Comment by scottcorgan** [comment]
@hchasey probably, but the Sass file seems super out of date. I updated the HTML one tho.

**Comment by scottcorgan** [comment]
@hchasey it's probably worth having @shanebo spend time on the CSS one. He's all about that documentation 👍

File: style/html/README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 HTML
 ====
 
+* Use `snake_case` for file names like `header_animation.es6`.
```
Comment (hchasey) [inline style/html/README.md]: The "like" example should be an html/erb file type.

**Comment by scottcorgan** [comment]
@hchasey fixed and back to you

**Pull request merged**